widget/entry_validation.go

```diff
@@ -47,6 +63,14 @@ func (e *Entry) SetValidationError(err error) {
 	}
 }
 
+// ShouldDisplayValidation returns true if the entry has not been interacted with
```

Comment:
The dirty check seems to be backwards here - if it is not dirty it should not display validation?